zhaquirks/tuya/ts0601_illuminance.py

```diff
@@ -0,0 +1,115 @@
+"""Tuya HOME-LUX illuminance sensor."""
+
+import math
+from typing import Dict
+
+from zigpy.profiles import zha
+from zigpy.quirks import CustomDevice
+import zigpy.types as t
+from zigpy.zcl.clusters.general import Basic, GreenPowerProxy, Groups, Ota, Scenes, Time
+from zigpy.zcl.clusters.measurement import IlluminanceMeasurement
+
+from zhaquirks.const import (
+    DEVICE_TYPE,
+    ENDPOINTS,
+    INPUT_CLUSTERS,
+    MODELS_INFO,
+    OUTPUT_CLUSTERS,
+    PROFILE_ID,
+)
+from zhaquirks.tuya import DPToAttributeMapping, TuyaLocalCluster, TuyaNewManufCluster
+
+TUYA_BRIGHTNESS_LEVEL_ATTR = 0x01  # 0-2 "Low, Medium, High"
+TUYA_ILLUMINANCE_ATTR = 0x02  # [0, 0, 3, 232] illuminance
```

Comment:
No problem with setting as constants, but I would prefer not to use the `_ATTR` sufix because it can be confusing (they are not clusters attributes).
Replace all around with a DataPoint nomenclature:
```python
TUYA_BRIGHTNESS_LEVEL_DP = 0x01  # 0-2 "Low, Medium, High"
TUYA_ILLUMINANCE_DP = 0x02  # [0, 0, 3, 232] illuminance
```